cli/tests/unit/flash_test.ts

```diff
@@ -58,31 +58,38 @@ Deno.test(async function httpServerCanResolveHostnames() {
 });
 
 Deno.test(async function httpServerRejectsOnAddrInUse() {
-  const ac = new AbortController();
+  const ac1 = new AbortController();
   const listeningPromise = deferred();
+  let port: number;
 
   const server = Deno.serve({
     handler: (_req) => new Response("ok"),
     hostname: "localhost",
-    port: 4501,
-    signal: ac.signal,
-    onListen: onListen(listeningPromise),
-    onError: createOnErrorCb(ac),
+    port: 0,
+    signal: ac1.signal,
+    onListen: (addr) => {
+      port = addr.port;
+      listeningPromise.resolve();
+    },
+    onError: createOnErrorCb(ac1),
```

Comment:
Using port number 4501 makes CI really flaky. On my local ubuntu machine I was not able to  see this flakiness though.